### PR TITLE
Use langfuse keys from os

### PR DESCRIPTION
**Pull request by Dev-Khant**
Contributes to https://github.com/langfuse/langfuse/issues/232

### CODE REVIEW

File: langfuse/client.py [hunk 1]
```diff
@@ -36,9 +36,6 @@ class Langfuse(object):
 
     def __init__(
         self,
-        public_key: str,
-        secret_key: str,
-        host: Optional[str] = None,
```
Comment (marcklingen) [inline langfuse/client.py:41]: For backwards compatibility, I'd suggest to keep the constructor arguments (public, secret, host) and make them optional. This applies to the SDK and the langchain callback handler. 

If set, constructor arguments then take precedence over environment variable. Removing the argument leads to unexpected behavior for existing users

**Review by marcklingen — CHANGES_REQUESTED** [review]
Thanks for your contribution on thi, see comment above regarding backwards compatibility

File: langfuse/client.py (hunk 1 above)
Comment (Dev-Khant) [reply langfuse/client.py:41]: Yes, it makes sense I have made the changes. And if access keys are not present, we are already showing a warning.

File: langfuse/callback.py [hunk 2]
```diff
@@ -34,14 +35,17 @@ def __init__(
...
+
         # If we're provided a stateful trace client directly
         if statefulTraceClient:
             self.trace = Run(statefulTraceClient, None)
             self.runs = {}
 
         # Otherwise, initialize stateless using the provided keys
         elif public_key and secret_key:
-            self.langfuse = Langfuse(public_key, secret_key, host, debug=debug, release=release)
+            self.langfuse = Langfuse(debug=debug, release=release)
```
Comment (maxdeichmann) [inline langfuse/callback.py:48]: Should we set the keys here?

Comment (Dev-Khant) [reply langfuse/callback.py:48]: Yes makes more sense above we are accessing it so here we can keep it

**Review by marcklingen — CHANGES_REQUESTED** [review]

File: langfuse/callback.py [hunk 3]
```diff
@@ -31,6 +32,9 @@ def __init__(
         statefulClient: Optional[StatefulTraceClient | StatefulSpanClient] = None,
         release: Optional[str] = None,
     ) -> None:
+        public_key = public_key if public_key else os.environ.get("LF_PK")
```
[the remark this answers is not in the document]
Comment (Dev-Khant) [reply langfuse/callback.py:35]: Makes more sense, didn't see it

File: tests/api_wrapper.py [hunk 4]
```diff
@@ -1,10 +1,13 @@
+import os
 import requests
 
 
 class LangfuseAPI:
-    def __init__(self, username=None, password=None, base_url="http://localhost:3000"):
-        self.auth = (username, password) if username and password else None
-        self.BASE_URL = base_url
+    def __init__(self, username=None, password=None, base_url=None):
```
[the remark this answers is not in the document]
Comment (marcklingen) [reply tests/api_wrapper.py:6]: Ok perfect, agree that we should not optimize for tests here

**Review by marcklingen — APPROVED** [review]

**Review by maxdeichmann — APPROVED** [review]

**Comment by maxdeichmann** [comment]
Thanks for the contribution @Dev-Khant

**Pull request merged**